src/nwb_datajoint/figurl_views/SpikeSortingRecordingView.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Union
+import datajoint as dj
+import sortingview as sv
+from sortingview.SpikeSortingView import create_raw_traces_plot
+from sortingview.SpikeSortingView.Figure import Figure
+import numpy as np
+import kachery_client as kc
+import os
+import spikeinterface as si
+
+from ..common.common_spikesorting import SpikeSortingRecording
+
+schema = dj.schema('figurl_view_spike_sorting_recording')
+
+@schema
+class SpikeSortingRecordingView(dj.Computed):
+    definition = """
+    # Schema for storing figurl views of spike sorting recordings
+    -> SpikeSortingRecording
+    ---
+    figurl: varchar(10000)
+    """
+    def make(self, key):
+        # Get the SpikeSortingRecording row
+        a = (SpikeSortingRecording & key).fetch1()
+        sort_group_id = a['sort_group_id']
+        sort_interval_name = a['sort_interval_name']
+        recording_id = a['recording_id']
+        recording_path = a['recording_path']
+
+        # Load the SI recording extractor
+        recording: si.BaseRecording = si.load_extractor(recording_path)
+        
+        # Raw traces (sample)
+        # Extract the first 1 second of traces
+        traces: np.array = recording.get_traces(
+            start_frame=0,
+            end_frame=int(recording.get_sampling_frequency() * 1)
+        ).astype(np.float32)
+        f1 = create_raw_traces_plot(
+            traces=traces,
+            start_time_sec=0,
+            sampling_frequency=recording.get_sampling_frequency(),
+            label='Raw traces (sample)'
+        )
+
+        # Electrode geometry
+        f2 = create_electrode_geometry(recording)
+
+        # Mountain layout
+        F = create_mountain_layout(
+            figures=[f1, f2],
+            label=f'{recording_id}:{sort_group_id}:{sort_interval_name}'
+        )
+
+        # Insert row into table
+        key2 = key
```

Comment:
I think unless you deepcopy here you're still mutating the original dictionary. 
<img width="505" alt="image" src="https://user-images.githubusercontent.com/8053989/156402983-919b370f-9556-4d47-b4c8-a50f953b3889.png">

Reply:
Now improved:

```python
key2 = dict(key, **{
    'figurl': F.url()
})
```